### DISCUSSION TITLE
ft.Video doesn’t allow you to move the slider

### DISCUSSION THREAD

**Discussion by 7576457** [flet-dev/flet#4234] [Q&A] [answered]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

The problem is that I can’t move the slider when I place the ft.VideoMedia widget inside an ft.Container. However, when I put it in a variable without the ft.Container, I can move the slider.

I’ve attached the video and code.
To reproduce the problem, first replace the current sample_media with this:
```python
sample_media = [
    ft.Container(
        content=ft.VideoMedia(
            "https://user-images.githubusercontent.com/28951144/229373720-14d69157-1a56-4a78-a2f4-d7a134d7c3e9.mp4",
        ),
    ),
]

```
Then change it to this:
```python
sample_media = [
    ft.VideoMedia(
        "https://user-images.githubusercontent.com/28951144/229373720-14d69157-1a56-4a78-a2f4-d7a134d7c3e9.mp4",
    ),
]
```

### Code sample

<details open><summary>Code</summary>

```python
import flet as ft

def main(page: ft.Page):
    sample_media = [
        ft.Container(
            content=ft.VideoMedia(
                "https://user-images.githubusercontent.com/28951144/229373720-14d69157-1a56-4a78-a2f4-d7a134d7c3e9.mp4",
            ),
        ),
    ]

    page.add(
        ft.Video(
            expand=True,
            playlist=sample_media[0:2],

        )
    )

ft.app(main)
```

</details>

### To reproduce

not have

### Screenshots / Videos

<details open>
<summary>Captures</summary>

https://github.com/user-attachments/assets/736568e7-2b9b-412a-9ac5-e5d7d5f9f4b4

</details>

### Operating System

Windows

### Operating system details

Windows 11

### Flet version

0.24.1

### Regression

No, it isn't

**Comment by 7576457** [comment]
I also tested this bug in version 0.25.0.dev3570, and the issue is still there

**Comment by syleishere** [comment]
Are you setting your container expand=True?

**Comment by ndonkoHenri** [comment, marked as answer]
Seems like you are doing it wrongly here:
```python
sample_media = [
    ft.Container(
        content=ft.VideoMedia(
            "https://user-images.githubusercontent.com/28951144/229373720-14d69157-1a56-4a78-a2f4-d7a134d7c3e9.mp4",
        ),
    ),
]

```

`Container.content` cannot be a `VideoMedia`, because `VideoMedia` isn't a Flet Control. 
Modern IDEs highlight such basic errors (Pycharm below):
![image](https://github.com/user-attachments/assets/e335f385-5e81-453c-baeb-3a1e02b7c537)
![image](https://github.com/user-attachments/assets/0fc635ec-d79d-425f-a490-55abfe366495)

This is the reason why your second `sample_media` works - that's the right way to do it:
```python
sample_media = [
    ft.VideoMedia(
        "https://user-images.githubusercontent.com/28951144/229373720-14d69157-1a56-4a78-a2f4-d7a134d7c3e9.mp4",
    ),
]
```

![image](https://github.com/user-attachments/assets/eff36fb3-4b5a-4bd1-ac87-88e8c95aa405)

**Comment by syleishere** [comment]
```
        sample_media = []
        for i in range(0, len(SONG_LIST)):
                sample_media.append(ft.VideoMedia(SONG_LIST[i]))
```

**Reply by 7576457** [reply]
what is it?

**Reply by syleishere** [reply]
Songlist? Just an array of urls
